app/src/main/java/br/com/dionataferraz/vendas/model/GetTransactionModel.kt

```diff
@@ -0,0 +1,8 @@
+package br.com.dionataferraz.vendas.model
+
+import java.util.Date
+
+data class GetTransactionModel(
+    val id: Number, val date: Date, val value: Double, val transactionType: String,
```

Comment:
Não usamos Number, aqui deveria ser um int e no caso do date sempre vai falhar a request, pois estamos esperando um transactionDate como name do atributo
```suggestion
    val id: Int, val date: Date, val value: Double, val transactionType: String,
```